### ISSUE TITLE
create nn_batch_norm3d module

### ISSUE THREAD

**Issue by dominikj2** [mlverse/torch#368]
hi @dfalbel,
R has nn_batch_norm2d and wonder when nn_batch_norm3d will be available? 

Could you please provide an R version of torch.nn.BatchNorm3d ?

I am interested to learn the difference between  nn_batch_norm2d  and nn_batch_norm1d and work out nn_batch_norm3d but I don't know how to see the source code for existing batch norms. 

I tried https://rdrr.io/cran/torch/man/nn_batch_norm1d.html

**Comment by dfalbel** [comment]
Hi @dominikj2 

It seems that nn_batch_norm3d is already implemented in the dev version..
See the definition here: https://github.com/mlverse/torch/blob/master/R/nn-batchnorm.R#L330

You can install the dev version with:

```
remotes::install_github("mlverse/torch")
```

If you are on Windows, make sure to have RTools installed.

**Comment by dominikj2** [comment]
Wonderful. Thank you!
